src/main/java/core/basesyntax/figure/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.Color;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double firstParallel;
+    private final double secondParallel;
+    private final double height;
+
+    public IsoscelesTrapezoid(
+            Color color,
+            double firstParallel,
+            double secondParallel,
+            double height
+    ) {
+        this.color = color;
+        this.firstParallel = firstParallel;
+        this.secondParallel = secondParallel;
+        this.height = height;
+    }
```

Comment:
Too many new lines reduces readability:
Something like that if you are limited by line length
```suggestion
    public IsoscelesTrapezoid(Color color, double firstParallel,
                            double secondParallel, double height) 
   {
        this.color = color;
        this.firstParallel = firstParallel;
        this.secondParallel = secondParallel;
        this.height = height;
    }
```